### PR TITLE
Fixes #23936 - disable NIC fact parsing on Windows

### PR DESCRIPTION
**Pull request by lzap**
I propose to disable NIC fact parsing on Windows family, it currently fails as format of Ansible facts on Windows is completely different.

We can implement this later but this should unblock users at least.

### CODE REVIEW

**Comment by evgeni** [comment]
I don't think it *fixes* the issue, but it's a nice workaround for now

**Comment by lzap** [comment]
Indeed, but that is our commit message pattern. I think it makes sense to associate this patch with the issue you've found to inform the users there is something going on in this regard.

Added tests and windows fact fixture.

**Comment by evgeni** [comment]
No, you can "Refs #23936" without Fixes.

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^8.8.0",
+    "@theforeman/vendor": "^8.15.1",
```
Comment (evgeni) [inline package.json]: the changes in this file look totally unrelated?!

**Comment by ezr-ondrej** [comment]
@evgeni wanna finish the review?

File: package.json (hunk 1 above)
Comment (lzap) [reply package.json]: Yup!

**Comment by lzap** [comment]
And rebased.

**Comment by evgeni** [comment]
For all I know, the diff looks sane.

But I still think the commit message should be `Refs #23936 - …`, not `Fixes …`

**Review by tbrisker — APPROVED** [review]
Thanks @lzap ! will fix the commit message on merge to refs.
